src/gui/src/gui.cpp

```diff
@@ -1129,6 +1129,9 @@ int startGui(int& argc,
   font.setPointSize(12);
   QApplication::setFont(font);
 
+  // Enable coalescing of high-frequency events
+  QCoreApplication::setAttribute(Qt::AA_CompressHighFrequencyEvents);
```

Comment:
doc says, `On the X11 windowing system, the default value is true`,  so I would expect no change...

https://doc.qt.io/qt-6/qt.html

Reply:
I'm using GWSL which I thought used Wayland, but maybe not. It was worth a shot because this is the only control I can find related to this issue.
I also tried `Qt::AA_CompressTabletEvents` because I thought maybe a laptop trackpad counts as that, but it didn't help either.